distributed-process-systest.cabal

```diff
@@ -22,7 +22,7 @@ library
                      ansi-terminal >= 0.5 && < 0.7,
                      binary >= 0.5 && < 0.9,
                      bytestring >= 0.9 && < 0.11,
-                     distributed-process >= 0.6.1 && < 0.7,
+                     distributed-process >= 0.6.1 && < 0.8,
                      distributed-static,
```

Comment:
https://github.com/haskell-distributed/distributed-process-systest/blob/master/distributed-process-systest.cabal#L25

This PR looks to be un-necessary now `distributed-process >= 0.6.1 && < 0.8,` already in master